examples/react/trimmed/src/useMosiacReactTable.tsx

```diff
@@ -0,0 +1,20 @@
+// This is an example of the React-specific hook that'd be used to allow
+// Mosaic's data to drive an instance of TanStack Table in an app.
+import * as React from 'react';
+import { createMosaicDataTableClient } from '@nozzle/mosaic-tanstack-table-core/trimmed';
+import { useStore } from '@tanstack/react-store';
+import type { MosaicDataTableOptions } from '@nozzle/mosaic-tanstack-table-core/trimmed';
+
+export function useMosaicReactTable<TData = unknown>(
```

Comment:
This'd essentially be what'll live in the `@nozzle/mosaic-tanstack-react-table` package. This package is what you as an end-user would `npm install`, not `core`.